### PR TITLE
add resource pack

### PR DESCRIPTION
**Pull request by electron271**
adds panorama and custom splash text

### CODE REVIEW

File: resourcepacks/hajpack/assets/minecraft/texts/splashes.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+trans rights are human rights!
+hajpack v1-alpha - welcome to buggy hell
+rm -rf /
```
Comment (RPMYT) [inline resourcepacks/hajpack/assets/minecraft/texts/splashes.txt:3]: you'd be surprised how many people would actually run this
even if it needs --no-preserve-root and sudo, maybe don't have this one

File: index.toml [hunk 2]
```diff
@@ -88,3 +88,35 @@ metafile = true
...
+[[files]]
+file = "resourcepacks/hajpack/assets/minecraft/textures/gui/title/background/panorama_1.png"
+hash = "a1c93e4cc4e6f73a63085351a83287a50f85c7ac636ce4fdeddd7951e3d71963"
+
+[[files]]
+file = "resourcepacks/hajpack/assets/minecraft/textures/gui/title/background/panorama_2.png"
+hash = "7800c9e5edca3a3fc7898cd25ef9694b019a21abebe5871d59a1e109a144f561"
+
+[[files]]
+file = "resourcepacks/hajpack/assets/minecraft/textures/gui/title/background/panorama_3.png"
```
Comment (RPMYT) [inline index.toml:109]: this layout is hhhhh

File: index.toml [hunk 3]
```diff
@@ -88,3 +88,35 @@ metafile = true
...
+[[files]]
+file = "resourcepacks/hajpack/assets/minecraft/texts/splashes.txt"
+hash = "257c868194c39558070bff039c671c7c465c13832293b3286658461cb3284fa3"
+
+[[files]]
+file = "resourcepacks/hajpack/assets/minecraft/textures/gui/title/background/panorama_0.png"
+hash = "88fe8a06db4dfeaf5607416d6a5bc399b6cbe0c0f6deaa4bd6e3d61b8f25cd19"
+
+[[files]]
+file = "resourcepacks/hajpack/assets/minecraft/textures/gui/title/background/panorama_1.png"
```
Comment (RPMYT) [inline index.toml:101]: this layout is hhhhh

**Review by RPMYT — CHANGES_REQUESTED** [review]

File: index.toml (hunk 2 above)
Comment (electron271) [reply index.toml:109]: thats just how resource packs are, blame mojang

File: resourcepacks/hajpack/assets/minecraft/texts/splashes.txt (hunk 1 above)
Comment (electron271) [reply resourcepacks/hajpack/assets/minecraft/texts/splashes.txt:3]: literally 1987 by jorge oil

File: index.toml (hunk 2 above)
Comment (RPMYT) [reply index.toml:109]: nah I mean the image itself

Comment (electron271) [reply index.toml:109]: resolved in discord convo

**Review by RPMYT — APPROVED** [review]

File: resourcepacks/hajpack/assets/minecraft/texts/splashes.txt [hunk 4]
```diff
@@ -1,6 +1,6 @@
 trans rights are human rights!
 hajpack v1-alpha - welcome to buggy hell
-rm -rf /
+you should NOT run rm -rf /
```
[the remark this answers is not in the document]
Comment (electron271) [reply resourcepacks/hajpack/assets/minecraft/texts/splashes.txt:3]: lmao

**Pull request merged**